wagtail/test/utils/wagtail_tests.py

```diff
@@ -4,14 +4,14 @@
 from bs4 import BeautifulSoup
 from django import VERSION as DJANGO_VERSION
 from django.contrib.auth import get_user_model
-from django.http import HttpRequest
+from django.http import HttpResponse
 from django.test.testcases import assert_and_parse_html
 
 
 class WagtailTestUtils:
     @staticmethod
-    def get_soup(request: HttpRequest, parser="html.parser") -> BeautifulSoup:
-        return BeautifulSoup(request.content, parser)
+    def get_soup(response: HttpResponse, parser="html.parser") -> BeautifulSoup:
```

Comment:
🤦 you're right, thanks.

I was actually planning to rebase #10326 and make that PR use this method, but I noticed that the tests modified in that PR use strings that don't come from an HttpResponse.

I was thinking we should change this method to take a string/bytestring and pass it directly to `BeautifulSoup`, so you can use it as `self.get_soup(response.content)` or `self.get_soup("<p>Some HTML string</p>")`

Reply:
I guess that works, even if it is slightly eating away at the benefits of `get_soup` as a shorthand over calling BeautifulSoup directly :-)